.github/workflows/integration-tests.yaml

```diff
@@ -23,7 +23,7 @@ jobs:
       - name: Set up Go
         uses: actions/setup-go@v3
         with:
-          go-version: "1.20"
+          go-version: "1.21"
```

Comment:
You could use, so we don't have to update it everytime
```
go-version-file: <go.mod file> 
```
https://github.com/aws/aws-network-policy-agent/blob/main/.github/actions/install-dependencies/action.yaml#L8